problems/ad_argon_chemistry/reaction_em + Ar -> em + Ar*.txt

```diff
@@ -0,0 +1,200 @@
+2.9270e-01 0.0000e+00 
```

Comment:
Are these spaces in the file name? Gross!

Reply:
Yeah, it's ugly...I don't like it either. It was just for convenience at first and I got stuck with it. I have an alternative where you specify the file name for each reaction but it hasn't been finalized - if you'd prefer I can finish that up quickly and replace those as part of this PR.

Reply:
It depends how much work it is. It would be nice to not have files with spaces in the names for too long in the repo

Reply:
It shouldn't be bad. The groundwork is already in place. I should have it done by COB tomorrow.

Reply:
@lindsayad Alright, I couldn't quite get PetscJacobianTester working well. I tried scaling the mesh which helped a tiny bit, but when I tried scaling the time units I got a different answer than I did with unscaled timesteps, so I don't really trust that. However, it did converge and the relative difference was about an order of magnitude lower. I've added an example with a pretty loose relative tolerance 1e-4 just so it'll pass (without any timestep scaling), but that doesn't need to be final.

You're right, scaling everything to near unity is a little difficult with this system of equations. I'm not sure what the best path forward is.

Reply:
Ok, 1e-4 isn't terrible.

Am I a crazy person or does this file not still have spaces in the name?

Reply:
Nope, you're not crazy. I'm just a fool. I changed all the filenames in the tests directory but not in problems. Give me a minute...

Reply:
Sorry about that. Should be fixed now.